microshift_getting-started/microshift-architecture.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: ASSEMBLY
+[id="microshift-architecture"]
+= Architecture
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-architecture
+
+toc::[]
+
+== The benefits of running containers on the edge
+
+{product-title} is a single-node container orchestration runtime designed for running applications in low-resource edge environments. Like {OCP}, {product-title} is based on Kubernetes and retains many of the benefits of using containers, bringing those benefits to the edge. Because edge and small-form-factor computing are by nature created with only the essentials needed to operate, so is {product-title}.
+
+== Key differences from {OCP}
+* Works offline and in low-connectivity environments
```

Comment:
This could use some clarification. We have [documentation](https://docs.openshift.com/container-platform/4.11/installing/installing_vsphere/installing-restricted-networks-installer-provisioned-vsphere.html) for installing in offline/highly restricted environments for OCP.